material-github.user.css

```diff
@@ -1080,8 +1080,282 @@
+		background-color: var(--bg5) !important;
+	}
+
+	/* Repo, Insights, Forks */
+	.network-tree {
+		width: 26px !important;
+		height: 0 !important;
+		padding-top: 22px !important;
+	}
+
+	/* Repo, Releases, Releases */
+	.release-main-section {
+		border-left: 2px solid var(--bg5)!important;
+	}
+	.select-menu-modal position-absolute text-md-left .SelectMenu-list {
+		background-color: var(--bg)!important;
+	}
+	.select-menu-modal position-absolute text-md-left .select-menu-list {
+		background-color: var(--bg);
+		border-top: 0px solid var(--bg5)!important;
+	}
+	.select-menu-text-filter:first-child:last-child {
+		padding-bottom: 10px;
+		border-bottom: 1px solid var(--bg5)!important;
+	}
+
+	/* Repo, Pull requests */
+	.merge-status-list {
+		border-top-color: var(--bg2);
+	}
+
+	.full-commit .commit-meta {
+		border-top-color: inherit;
+	}
+
+	.clearfix.js-issues-results .tabnav,
+	.discussion-sidebar-item.sidebar-notifications,
+	div#partial-users-participants {
+		border-color: var(--bg5) !important;
+	}
+
+	.timeline-comment-group table.d-block {
+		visibility: initial !important;
+		display: initial !important;
+	}
+
+	/* Repo, Actions */
+	.bg-gray-dark {
+		background-color: var(--bg1)!important;
+	}
+	.CheckRun-header {
+		border-top: 1px solid var(--bg2)!important;
+		border-bottom: none;
+	}
+	.CheckRun {
+		background-color: var(--bg3)!important;
+	}
+
+	/* Header, Sourcegraph - Chrome Exetnsion */
+	.command-list,
+	.show.chromed-list-popover.Box .popvoer-inner .command-list,
+	.flex-items-center.jump-to-suggestions-path.p-2 {
+		background-color: var(--bg1) !important;
+	}
+	.show.command-list-popover.Box {
+		border-color: var(--bg5)!important;
+	}
+	.navigation-focus .flex-auto {
+		color: var(--selected) !important;
+	}
+
+	/* Github Hovercard - Chrome Extension */
+	.ghh-theme-github,
+	.tooltipster-box,
+	.tooltipster-top,
+	.tooltipster-bottom,
+	.tooltipster-content,
+	.tooltipster-arrow-border,
+	.tooltipster-arrow-background {
+		border-color: var(--bg4) !important;
+		background-color: var(--bg4) !important;
+		color: rgba(255, 255, 255, 0.7) !important;
+	}
+
+	.ghh-stats {
+		border-color: var(--bg4) !important;
+	}
+
+	.ghh-stats span {
+		color: rgba(255, 255, 255, 0.7) !important;
+	}
+
+	.ghh-stats > a + a {
+		border-left: 1px solid var(--bg6) !important;
+	}
+
+	.ghh-state-open {
+		color: rgba(44, 190, 78)!important;
+		border-color: var(--bg) !important;
+		background-color: var(--bg4) !important;
+	}
+
+	.ghh-state-verified {
+		color: rgba(34, 134, 58) !important;
+		border-color: var(--bg) !important;
+		background-color: var(--bg3) !important;
+	}
+
+	.ghh-markdown-separator,
+	.ghh-more-separator {
+		border-bottom: 1px solid var(--bg6) !important;
+	}
+
+	.ghh-issue-body {
+		background-color: var(--bg2)!important;
+		color: rgba(255, 255, 255, 0.65) !important;
+	}
+
+	.ghh-readme pre,
+	.ghh-issue-body pre {
+		background: var(--bg4) !important;
+		color: rgba(204,204,204) !important;

```

Comment:
rgba should take 4 parameters, I think you could set it to `rgba(255,255,255, .8)`

Reply:
actually, please set it to `color: rgba(255, 255, 255, calc(0.8 * /*[[text_brightness]]*/)) !important;`
This would make it compatible with text brightness selector.
Thanks!

Reply:
This was omissive last night.
I would check if have rgba without taking the 4th parameters as soon as possible.